src/pykiso/test_coordinator/test_execution.py

```diff
@@ -323,6 +324,23 @@ def collect_test_suites(
     return list_of_test_suites
 
 
+def abort(reason: str = None) -> None:
+    """Quit ITF test and log an error if a reason is indicated and
+    if any errors occurred it logs them.
+
+    :param reason: reason to abort, defaults to None
+    """
+    if reason:
+        log.critical(reason)
+    log.exception("Tests were aborted because of the following error :")
```

Comment:
This should be removed IMO, as nothing is raised this will just be shown as a simple error message that doesn't really add value

Reply:
Done